Public/Invoke-DockerTag.ps1

```diff
@@ -0,0 +1,20 @@
+#Requires -PSEdition Core -Version 6
+
+function Invoke-DockerTag {
+    [CmdletBinding()]
+    param (
+        [Parameter(mandatory=$true)]
+        [String]
+        $SourceImage,
+        [ValidateNotNullOrEmpty()]
+        [String]
```

Comment:
I am not sure this even works as expected. I think you should add test cases where you try setting them to null or an empty string to check.

Reply:
🌞